app/profile.py

```
@@ -62,152 +62,74 @@ def make_card(status, count):
-    from starlette.responses import JSONResponse
-    import json
-
-    body = await req.body()
-    data = json.loads(body)
-    item_ids = data.get("item_ids", [])
-    status = data.get("status")
-
-    if not item_ids or not status:
-        return JSONResponse({"error": "Missing item_ids or status"}, status_code=400)
+@profile_app.post("/bulk/set_status")
+async def bulk_set_status(req: Request, auth, sess, status: str, status_filter: str = None, page: int = 1):
```

Ruff (0.14.10)

`Request` may be undefined, or defined from star imports

(F405)